R/volcanoPlot.R

```diff
@@ -14,18 +13,19 @@
 #'   stratification_col="AEBODSYS",
 #'   group_col="ARM",
 #'   reference_group="Placebo",
-#'   comparison_group="Xanomeline High Dose",
+#'   comparison_group=c("Xanomeline High Dose","Xanomeline Low Dose"),
 #'   id_col="USUBJID"
 #' )
 #' stats<-getStats(dfAE=safetyData::adam_adae, dfDemog = safetyData::adam_adsl, settings)
 #' volcanoPlot(stats)
 #'
 #' @import ggplot2
 #' @importFrom plotly ggplotly
+#' @import crosstalk
```

Comment:
```suggestion
#' @importFrom crosstalk SharedData bscols
```

Reply:
Only importing what we need will help us to avoid the following warning:

```Warning message:
replacing previous import ‘crosstalk::getDefaultReactiveDomain’ by ‘shiny::getDefaultReactiveDomain’ when loading ‘volcanoPlot’ ```